packages/perseus/src/widgets/number-line/number-line.tsx

```diff
@@ -261,7 +261,7 @@ class NumberLine extends React.Component<Props, State> implements Widget {
 
     isValid: () => boolean = () => {
         const range = this.props.range;
-        let initialX = this.props.userInput.numLinePosition;
+        let initialX = this.props.userInput?.numLinePosition;
```

Comment:
I used chaining here, but I'm wondering if we need to be setting defaults. 🤔

Reply:
Maybe we need to handle it better in `getStartUserInput` for NumberLine; or is this something that we're seeing in the editing experience?